src/components/LoginSignUp/AuthenticationService.js

```diff
@@ -101,6 +102,7 @@ const getJWTToken = async ( allowAnonymousJWTToken: boolean = false ): Promise<?
 
   if (
     !jwtToken
+    || !jwtTokenExpiration
```

Comment:
flow wasn't happy with using an arithmetic operator with `jwtTokenExpiration` (which it thinks could be `void`) on the next line